bw2color.py

```diff
@@ -0,0 +1,53 @@
+from matplotlib import pyplot as plt
+from keras.preprocessing.image import ImageDataGenerator, array_to_img, img_to_array, load_img
+from skimage.color import rgb2lab, lab2rgb, rgb2gray
+from keras.models import model_from_json
+from skimage.io import imsave
+import numpy as np
+import tensorflow as tf
+import cv2
+import os
+import argparse
+import sys
+from tqdm import tqdm as tqdm
+
+ap = argparse.ArgumentParser()
+ap.add_argument("-i", "--image", required=True, help="path to input image")
+args = vars(ap.parse_args())
+
+# load json and create model
+
+json_file = open('model.json', 'r')
+loaded_model_json = json_file.read()
+json_file.close()
+loaded_model = model_from_json(loaded_model_json)
+
+# load weights into new model
+loaded_model.load_weights("model.h5")
+
+if args["image"] is None:
+    print("Can't read the image file." + "\nPlease make sure you are passing a valid path and it points to an image.")
+    sys.exit()
+for j in tqdm(range(1, 10), desc="Generating"):
```

Comment:
It would be nice if you could put comments corresponding to the lines inside the interaction block below so that even beginners can follow

Short phrases like, "normalize image array", etc